Sparkle/SUUpdater.m

```diff
@@ -317,18 +315,7 @@ - (void)checkForUpdatesInBackground
     }
 */
     // Do not use reachability for a preflight check. This can be deceptive and a bad idea. Apple does not recommend doing it.
-    SUUpdateDriver *theUpdateDriver = [(SUBasicUpdateDriver *)[[SUAutomaticUpdateDriver class] alloc] initWithUpdater:self];
-
-    if(!automatic)
-    {
-        // Notify host app that update has aborted
-        if (self.delegate && [self.delegate respondsToSelector:@selector(updater:didAbortWithError:)])
-        {
-            [self.delegate updater:self didAbortWithError:
-             [NSError errorWithDomain:SUSparkleErrorDomain code:SUTemporaryDirectoryError userInfo:nil]];
-        }
-        return;
-    }
+   SUUpdateDriver *theUpdateDriver = [(SUBasicUpdateDriver *)[(automatic ? [SUAutomaticUpdateDriver class] : [SUScheduledUpdateDriver class])alloc] initWithUpdater:self];
```

Comment:
Oli, I've kept this line in as it was part of your original changeset, but I'm unsure of what it does. Could you clarify it (or mention if it's worth undoing?)

Reply:
Yup thats good - I don't know why this was removed as this would cause the code to crash. I believe I was basically in the process of writing in that file when I got pulled out of the user story and committed everything.

One thing though - I'd investigate using the forked implementation (line in red) vs this one. I believe Antoine might've had a good reason to ditch the `SUScheduledUpdateDriver` and remove it from the ternary if.